### ISSUE TITLE
loki-stack helm charts is deprecated

### ISSUE THREAD

**Issue by mjtrangoni** [fluxcd/flux2-monitoring-example#19]
Hi,

First of all, as a newbie here, thanks for the whole project!
I wanted to report that *loki-stack* Helm chart seems to be "deprecated" by grafana. See [here](https://community.grafana.com/t/difference-between-helm-charts-loki-and-loki-stack/87380/2).
So I guess it should be replaced by the Monolithic mode one.

Thank you!

Mario

**Comment by mjtrangoni** [comment]
@darkowlzz yes, I would love to take it! The only thing is that I will be on vacation until the 8th of January.

**Comment by kingdonb** [comment]
Thank you for contributing this here, I think it's fixed now in #47 - and up-to-date on all Prometheus stack chart versions and Loki, Promtail 🎉
